### DISCUSSION TITLE
How do I show Piggy bank in my dashboard graph

### DISCUSSION THREAD

**Discussion by Sebekerga** [firefly-iii/firefly-iii#6761] [Firefly III]
My bank doesn't support multiple accounts, and I'm currently not able to open account in other banks, meaning I'm stuck with a single account for all of my money (aside from cash). That includes my "Just in case" Piggy bank. When I open dashboard I'd like to see a graph of my Piggy bank balance and/or my main Account balance excluding money in a Piggy bank. 

How do I achieve such thing?

Also, I understand that I can create virtual account in Firefly and use it as my Piggy bank for savings, but in that case I'd like to see on a dashboard a total of my virtual savings and main account, basically actual balance on my debit card.

**Comment by JC5** [comment]
I'm sorry, but Firefly III isn't built for this. Although piggy banks are listed on the home page, they aren't available in graph form.

**Reply by Sebekerga** [reply]
Ah, unfortunate. Can you suggest any work arounds that I can use to achieve, maybe, not a graph, but perhaps just a number that would show me my money that aren't in a Piggy Bank?
